src/utils/institutions-helpers.ts

```diff
@@ -23,15 +25,32 @@ export const getMostSpecificUnit = (values: FormikInstitutionUnit): InstitutionU
 };
 
 // Find distinct unit URIs for a set of contributors' affiliations
-export const getDistinctContributorUnits = (contributors: Contributor[]): string[] => [
-  ...new Set(
-    contributors
-      .map((contributor) => contributor.affiliations)
-      .flat()
-      .filter((unit) => unit)
-      .map((unit) => unit.id)
-  ),
-];
+export const getDistinctContributorUnits = (contributors: Contributor[]) => {
+  const unitIds = contributors
+    .flatMap((contributor) => contributor.affiliations)
+    .filter((affiliation) => affiliation?.id)
+    .map((unit) => unit.id) as string[];
```

Comment:
aner ikke hvorfor det ikke virker uten `as string[]` her, for vi vet jo at det er det. funker heller ikke om jeg la til en `filter(x=>x)` etter map, men den insisterte likevel på at det ble typen `(string | undefined)[]` 😢